virtcontainers/sandbox.go

```diff
@@ -535,6 +535,31 @@ func (s *Sandbox) Release() error {
 	return s.agent.disconnect()
 }
 
+// Status gets the status of the sandbox
+func (s *Sandbox) Status() SandboxStatus {
+	var contStatusList []ContainerStatus
+	for _, c := range s.containers {
```

Comment:
The loop getting all the container statuses should leverage the function:
```go
func (s *Sandbox) StatusContainer(containerID string) (ContainerStatus, error)
```
from the previous commit. This would ensure that we also get some updated status for each container that needs some update.
Also this would change slightly the current function prototype, by adding an error return:
```go
func (s *Sandbox) Status() (SandboxStatus, error) {
```

Reply:
For the same reason as `StatusContainer()`, I want to fix it properly in a separate PR.

Reply:
Same here, please add a temporary comment ;)